packages/experiments/src/components/Stack/Stack.types.ts

```diff
@@ -2,27 +2,96 @@ import { IStyle } from '../../Styling';
 import { IStyleableComponent } from '../../Foundation';
 
 export interface IStackProps extends IStyleableComponent<IStackProps, IStackStyles> {
+  /**
+   * How to render the Stack.
+   */
   renderAs?: string | React.ReactType<IStackProps>;
+
+  /**
+   * CSS class name used to style the Stack.
+   */
   className?: string;
 
-  fill?: boolean;
-  collapseItems?: boolean;
+  /**
+   * Whether to render Stack child elements horizontally.
+   */
+  horizontal?: boolean;
+
+  /**
+   * How to align Stack child elements horizontally (along the x-axis).
+   */
+  horizontalAlignment?:
+    | 'start'
+    | 'end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'baseline'
+    | 'stretch';
+
+  /**
+   * How to align Stack child elements vertically (along the y-axis).
+   */
+  verticalAlignment?:
+    | 'start'
+    | 'end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'baseline'
+    | 'stretch';
```

Comment:
Could centrally define an Alignment type and then use it here to remove the duplication.

Reply:
My only concern is that this makes the documentation on the website harder to read - users won't know what `Alignment` is and so they won't know what values are acceptable to pass for `horizontalAlignment` and `verticalAlignment`. Unless there's a way to get the `Alignment` values into the documentation?

Reply:
If you declare a type of `IAutomation` in this file it should show up on the website.. at least that's happening with a bunch of the other types such as StyleProps and enums and such.

Reply:
I declared the `Alignment` type in this file like this:

`type Alignment =
  | 'start'
  | 'end'
  | 'center'
  | 'space-between'
  | 'space-around'
  | 'space-evenly'
  | 'baseline'
  | 'stretch';`

but it's not showing up on the website. Maybe it only works for interfaces and enums?

Reply:
Does it show up if you export it? It may be set to ignore private declarations.

Reply:
It does only work for interface and enums. It does not look for types atm.